### ISSUE TITLE
Make price difference column easier to parse visually

### ISSUE THREAD

**Issue by pepoluan** [endless-sky/endless-sky#9203]
### Problem Description

Currently, the price difference column in the Job Board is a bit hard to parse; not only it uses a proportional-spaced font, there's additional clutter due to parens:

![image](https://github.com/endless-sky/endless-sky/assets/642259/35fe8116-46ff-44d3-bc17-0225f4859fce)

### Related Issue Links

No related issue

### Desired Solution

If possible, use a monospaced font instead, right-aligned. Dropping the parens `(..)` will save horizontal space lost due to the possibly wider characters.

So the above table will look like this (rough mockup; I'm using Consolas as an example):

![mockup-price-diff](https://github.com/endless-sky/endless-sky/assets/642259/9ff06fa9-d5dd-4ef3-8c00-b8949b1c83fb)

Definitely easier to parse, and even less complicated because there are no parens (that do not serve any useful purpose anyways)

### Alternative Approaches

If it's too hard to use different fonts within one line there, at the very least please just drop the parens. They serve no purpose, and it's much easier to just scan for a "+" as the leftmost character in the price difference column.

**Comment by Just-Existing** [comment]
The lack of parenthesis and misfitting font actually make it harder for me to read.
The main reason I find it hard to read the trade prices has more to do with the fact that they are not ordered by price.
Here is what that might look like:
![Trade Prices sorted by price](https://github.com/endless-sky/endless-sky/assets/69060286/70cfa8e6-dfac-4c91-8e56-88f2f143f225)

**Comment by danaris** [comment]
Currently, ES can only use one font, period, with two set sizes: it uses essentially a spritesheet image of letters to draw all text.

#8572 would basically require changing this, so there's hope for progress, but it's not a trivial change.

**Comment by Hecter94** [comment]
ref: https://github.com/endless-sky/endless-sky/issues/2671
particularly: https://github.com/endless-sky/endless-sky/issues/2671#issuecomment-311162669

**Comment by Just-Existing** [comment]
I do not like the idea of adding more fonts. The game has very good visual continuity and another font might change that.

**Comment by jn64** [comment]
This does not require a monospace font or a different font family. Proportional fonts can support "monospaced" numbers (aka [tabular figures](https://practicaltypography.com/alternate-figures.html#tabular-and-proportional-figures)) specifically for the purpose of laying out numbers in columns like this.

The Ubuntu font supports it via an OpenType feature (although it seems to be implemented as _disabling_ proportional figures rather than _enabling_ tabular figures):

![Screenshot of the Ubuntu font with proportional and non-proportional figures](https://github.com/endless-sky/endless-sky/assets/23169302/fcc3b490-6696-44d9-b2b9-713972032c1a)

I understand this project uses a rasterised image of the font, so it's not as simple as flipping a switch on the OpenType feature, but the numbers you need already exist in the font you want.

---

Another problem is that the [hyphen-minus](https://en.wikipedia.org/wiki/Hyphen-minus) `-` and plus `+` symbols are not the same width as each other.

Again the Ubuntu font has the character you need. The "true" minus symbol in Ubuntu `−` is the same width as the plus _and_ tabular figures. So I would suggest having 2 separate glyphs for hyphen-minus (proportional, used to hyphenate text) and minus (used for negative numbers).

![Screenshot of the Ubuntu font with hyphen-minus and minus symbols](https://github.com/endless-sky/endless-sky/assets/23169302/bc8950d4-d7a4-4cdd-884d-fff7265f13c4)

---

I also recommend dropping the parentheses because they don't serve any function and take up precious space. The numbers are demarcated from the names by being right-aligned. If further visual separation is needed, I suggest drawing a vertical grid line between the 2 columns (rather than the unintentional squiggly line formed by the left parentheses).

**Comment by pepoluan** [comment]
> I also recommend dropping the parentheses because they don't serve any function and take up precious space. The numbers are demarcated from the names by being right-aligned. If further visual separation is needed, I suggest drawing a vertical grid line between the 2 columns (rather than the unintentional squiggly line formed by the left parentheses).

Yes! This please. The parens are cluttery, and because it's bigger visually the left parenthesis 'covers' the sign. Making scanning the number column needs more effort.

**Comment by Spectre84Div6** [comment]
As a temporary assistance to this could you just add 1 or 2 spaces between the + and - that way it might be slightly easier to see?

**Comment by xobes** [comment]
I had thought to colorize this.... thoughts?

**Comment by Amazinite** [comment]
#10151

**Comment by Amazinite** [comment]
Related: #3964
